docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,175 @@
+- [DCOS CLI](https://docs.mesosphere.com/install/cli/) installed locally
+
+
+## Install Kubernetes
+
+1. Configure and validate the Mesosphere Multiverse as a package source repository
+
+  ```
+  $ dcos config prepend package.sources https://github.com/mesosphere/multiverse/archive/version-1.x.zip
+  $ dcos package update --validate
+  ```
+
+2. Install Etcd
+
+  By default, the Kubernetes DCOS package comes with its own single-node Etcd, but in order to survive Kubernetes component process and/or container loss, an HA Etcd cluster is required.
+  Since [Etcd](https://github.com/mesosphere/etcd-mesos) can be installed on and managed by DCOS, it is the logical choice.
+
+  ```
+  $ dcos package install etcd
+  ```
+
+3. Verify that Etcd is installed and healthy
+
+  ```
+  $ dcos marathon app list
+  ```
+
+  The Etcd cluster takes a short while to deploy. Wait until it is healthy before proceeding to the next step.
+
+4. Create Kubernetes installation configuration
+
+  The Kubernetes installer must be configured to use the HA Etcd.
+
+  ```
+  $ cat >/tmp/options.json <<EOF
+  {
+    "kubernetes": {
+      "etcd-mesos-framework-name": "etcd"
+    }
+  }
+  EOF
+  ```
+
+5. Install Kubernetes
+
+  ```
+  $ dcos package install --options=/tmp/options.json kubernetes
+  ```
+
+6. Verify that Kubernetes is installed and healthy
+
+  ```
+  $ dcos marathon app list
+  ```
+
+  The Kubernetes cluster takes a short while to deploy. Wait until it is healthy before proceeding to the next step.
+
+7. Verify that Kube-DNS & Kube-UI are deployed, running, and ready
+
+  ```
+  $ dcos kubectl get pods --namespace=kube-system
+  NAME                READY     STATUS    RESTARTS   AGE
+  kube-dns-v8-tjxk9   4/4       Running   0          1m
+  kube-ui-v2-tjq7b    1/1       Running   0          1m
+  ```
+
+  This uses the DCOS-CLI Kubectl plugin, which is automatically downloaded and configured on first use.
+  Names and ages may vary.
+
+
+## Operations
+
+Explore the following resources for more information about usage of Kubernetes, Kubernetes on Mesos/DCOS, and DCOS itself.
+
+- [Kubernetes Examples](../../examples/README.md)
+- [Kubernetes on Mesos Documentation](../../contrib/mesos/README.md)
+- [Kubernetes on Mesos Release Notes](https://github.com/mesosphere/kubernetes-mesos/releases)
+- [Managing DCOS Services](https://docs.mesosphere.com/services/kubernetes/)
+- [DCOS Overview](https://docs.mesosphere.com/)
+
+
+## Uninstall Kubernetes
+
+1. Stop and delete all replication controllers and pods in each namespace:
+
+  Before uninstalling Kubernetes, destroy all the pods and replication controllers. The uninstall process will try to do this itself, but by default it times out quickly and may leave your cluster in a dirty state. This timeout can be configured in the Kubernetes installation configuration.
```

Comment:
Given your answer to my question about this, I think you can delete this last sentence. It implies a user can fix the problem by extending the timeout, but they can't. So it seems distracting to mention it here.